projects/js-packages/publicize-components/src/components/media-section/index.js

```diff
@@ -118,92 +136,96 @@ export default function MediaSection() {
-					{ mediaObject && renderPreview() }
-					{ ! mediaObject && ( attachedMedia.length ? <Spinner /> : ADD_MEDIA_LABEL ) }
-				</Button>
+				{ ! attachedMedia.length ? (
+					<>
+						<Button
+							variant="secondary"
+							size="small"
+							className={ mediaObject && styles.preview }
+							onClick={ open }
+						>
+							{ ! attachedMedia.length && ADD_MEDIA_LABEL }
+						</Button>
+						<span>{ __( 'Add an image or video', 'jetpack' ) }</span>
+					</>
+				) : (
+					<Spinner />
+				) }
 			</div>
 		),
-		[ mediaObject, attachedMedia, renderPreview ]
+		[ mediaObject, attachedMedia ]
 	);
 
-	const replaceMediaRender = useCallback(
-		( { open } ) => (
-			<Button onClick={ open } variant="secondary">
-				{ REPLACE_MEDIA_LABEL }
-			</Button>
-		),
-		[]
+	const setMediaRender = useCallback(
+		( { open } ) => ( mediaObject ? renderPreview( open ) : renderPicker( open ) ),
+		[ mediaObject, renderPreview, renderPicker ]
 	);
 
 	const onDismissClick = useCallback( () => setValidationError( null ), [] );
 
 	return (
-		<div className={ styles.wrapper }>
-			<MediaUploadCheck>
-				<MediaUpload
-					title={ ADD_MEDIA_LABEL }
-					onSelect={ onUpdateMedia }
-					allowedTypes={ allowedMediaTypes }
-					render={ setMediaRender }
-					value={ attachedMedia[ 0 ]?.id }
-				/>
-				{ mediaObject && (
-					<>
-						<MediaUpload
-							title={ REPLACE_MEDIA_LABEL }
-							onSelect={ onUpdateMedia }
-							allowedTypes={ allowedMediaTypes }
-							render={ replaceMediaRender }
-						/>
-						<Button onClick={ onRemoveMedia } variant="link" isDestructive>
-							{ REMOVE_MEDIA_LABEL }
-						</Button>
-					</>
-				) }
-				{ validationError && (
-					<Notice
-						className={ styles.notice }
-						isDismissible={ true }
-						onDismiss={ onDismissClick }
-						status="warning"
-					>
-						<p>{ validationErrorMessages[ validationError ] }</p>
-					</Notice>
-				) }
-				{ ! mediaObject && (
-					<Fragment>
-						<Text variant="title-small">{ __( 'Max image size', 'jetpack' ) }</Text>
-						<Notice className={ styles.max_notice } isDismissible={ false } status="info">
-							<Text>{ ` ${ maxImageSize } Mb` }</Text>
-						</Notice>
-						<Text variant="title-small">{ __( 'Allowed types', 'jetpack' ) }</Text>
-						<Notice className={ styles.max_notice } isDismissible={ false } status="info">
-							<Text>{ ` ${ allowedMediaTypes.join( ', ' ) }` }</Text>
+		<ThemeProvider>
+			<BaseControl label={ __( 'Media', 'jetpack' ) } className={ styles.wrapper }>
+				<MediaUploadCheck>
+					<p className={ styles.subtitle }>
+						{ __( 'Choose a visual to accompany your post.', 'jetpack' ) }
+					</p>
+					<MediaUpload
+						title={ ADD_MEDIA_LABEL }
+						onSelect={ onUpdateMedia }
+						allowedTypes={ allowedMediaTypes }
+						render={ setMediaRender }
+						value={ attachedMedia[ 0 ]?.id }
+					/>
+					<ExternalLink href={ getRedirectUrl( 'jetpack-social-media-support-information' ) }>
```

Comment:
The redirect url just takes me to MC now, do we plan to add it later?

Reply:
Yeah, that's a separate task: 1202411927350212-as-1203767773752680/f. Good catch!

Reply:
I came here to say the same thing. Now that this has been merged, I would recommend enabling the redirect as quickly as possible.